src/frontend/src/binder/expr/function/mod.rs

```diff
@@ -321,6 +321,15 @@ impl Binder {
                 self.ensure_table_function_allowed()?;
                 return Ok(TableFunction::new_file_scan(args)?.into());
             }
+            // `postgres_query` table function
+            if func_name.eq("postgres_query") {
+                reject_syntax!(
+                    arg_list.variadic,
+                    "`VARIADIC` is not allowed in table function call"
+                );
+                self.ensure_table_function_allowed()?;
+                return Ok(TableFunction::new_postgres_query(args)?.into());
```

Comment:
```suggestion
                return Ok(TableFunction::new_postgres_query(args).context("postgres_query error")?.into());
```

Tried a few error cases and the original error message was not very informative:
```
dev=> select * from postgres_query('localhos', '5432', 'postgres', 'postgres', 'test', 'select 2147483647/0 from t');
ERROR:  Failed to run the query

Caused by these errors (recent errors listed first):
  1: error connecting to server
  2: failed to lookup address information: nodename nor servname provided, or not known

dev=> select * from postgres_query('localhost', '543', 'postgres', 'postgres', 'test', 'select 2147483647/0 from t');
ERROR:  Failed to run the query

Caused by these errors (recent errors listed first):
  1: error connecting to server
  2: Connection refused (os error 61)
```